sdk/src/main/java/io/dapr/config/Properties.java

```diff
@@ -254,6 +254,24 @@ public class Properties {
           "DAPR_HTTP_CLIENT_MAX_IDLE_CONNECTIONS",
           DEFAULT_HTTP_CLIENT_MAX_IDLE_CONNECTIONS);
 
+
+
+  /**
+   * Dapr's default maximum inbound message size for GRPC in bytes.
+   */
+  public static final Property<Integer> GRPC_MAX_INBOUND_MESSAGE_SIZE_BYTES = new IntegerProperty(

```

Comment:
Can you rebase your PR? These changes shouldnt show up in your diff bc they are already merged into master

Reply:
it was in draft and outdated, now it is ready for review